packages/eas-cli/src/metadata/config/validate.ts

```diff
@@ -0,0 +1,50 @@
+import { appleRules } from '../apple/rules';
+import { createValidator, getReadableErrors } from '../utils/ajv';
+import { Issue, IssueRule, IssueSeverity } from './issue';
+import { MetadataConfig } from './schema';
+
+/** The pre-compiled AJV validator instance, with the metadata spec */
+const validator = createValidator().compile(require('../../../schema/metadata-0.json'));
```

Comment:
why does this need to be global? I'm not sure how heavy this part is but by doing it here you are adding overhead to every command in cli. If I understand correctly oclif does not lazy load commands, so this code will run every time.(I might be wrong about it)

Reply:
You are right, it doesn't need to be. AJV does recommend it, but I think that's mostly for backends. Will move it to the function instead